### PR TITLE
feat: rename bertychat -> bertymessenger

### PR DESCRIPTION
**Pull request by moul**
This PR:
* renames bertychat->bertymessenger in `go/`, `docs/`, `README.md`
* patches Makefiles in `js/` but does not change the rest of the `js/` files

### CODE REVIEW

File: js/packages/codegen/Makefile [hunk 1]
```diff
@@ -19,34 +19,34 @@ codegen_go_path := $(abspath $(berty_root)/vendor)
...
-	$(codegen_berty_chat_protos) \
+	$(codegen_berty_messenger_protos) \
 	$(codegen_berty_types_protos) \
 
 codegen_berty_protocol_sources := $(shell \
 	find $(abspath $(PWD)/..) -type f -name '*.hbs' -path '*/*/protocol/*' -not -path '*/node_modules/*')
-codegen_berty_chat_sources := $(shell \
-	find $(abspath $(PWD)/..) -type f -name '*.hbs' -path '*/*/chat/*' -not -path '*/node_modules/*')
+codegen_berty_messenger_sources := $(shell \
+	find $(abspath $(PWD)/..) -type f -name '*.hbs' -path '*/*/messenger/*' -not -path '*/node_modules/*')
```
[the remark this answers is not in the document]
Comment (n0izn0iz) [reply js/packages/codegen/Makefile:36]: I can do the dir changes on this PR if you want

[the remark this answers is not in the document]
Comment (n0izn0iz) [reply js/packages/codegen/Makefile:36]: actually forget about this, I can just create a `messenger` dir and import from it in the meanwhile

**Comment by moul** [comment]
hum, can you still make it in this PR so this PR won't brake the build?

btw, is it normal that CI is green?

**Comment by n0izn0iz** [comment]
the PR doesn't break the build, it's normal that the CI is green, I got confused my bad, we remove all `*/chat/*.hbs` files

**Pull request merged**